main/webapp/modules/core/scripts/reconciliation/recon-manager.js

```diff
@@ -120,6 +120,67 @@ ReconciliationManager.registerStandardService = function(url, f, silent) {
   });
 };
 
+ReconciliationManager.editStandardService = function(url, oldUrl, level, f, silent) {
+    DialogSystem.dismissUntil(level - 1);
+
+    var dismissBusy = function() {};
+    if (!silent) {
+        dismissBusy =  DialogSystem.showBusy();
+    }
+    var editService = function(data, mode) {
+        data.url = url;
+        data.ui = {
+            "handler" : "ReconStandardServicePanel",
+            "access" : mode
+        };
+
+        for (var i = 0; i < ReconciliationManager.customServices.length; i++) {
+            if (ReconciliationManager.customServices[i].url === oldUrl) {
+                ReconciliationManager.customServices[i] = data
+                break;
+            }
+        }
+        for (var i = 0; i < ReconciliationManager.standardServices.length; i++) {
+            if (ReconciliationManager.standardServices[i].url === oldUrl) {
+                ReconciliationManager.standardServices[i] = data
+                break;
+            }
+        }
+        ReconciliationManager._rebuildMap();
+        ReconciliationManager.save(f);
+        dismissBusy();
+    };
+
+    // First, try with CORS (default "json" dataType)
```

Comment:
As mentioned earlier, I would like that we do not duplicate this logic with the addition of a new service.